### ISSUE TITLE
Rock organ sounds like strings section sometimes

### ISSUE THREAD

**Issue by freq-mod** [sneakernets/DMXOPL#39]
Somewhat similar to #18 - rock organ patch is so "harsh" that it sounds like some kind of french horn or even tremolo strings ensemble (I even made an experiment swapping strings patch with rock "organ" one, it didn't sound that bad o.O)

There might be no way to solve this (at least I can't), but maybe...

P.S. If it helps a little - here is how I made rock organ in my OPN2 bank. Sounds rather hollow, though.
![zrzut ekranu z 2018-02-22 19-50-24](https://user-images.githubusercontent.com/32672779/36557878-c1dd5fd2-1809-11e8-9e27-0359c37156e8.png)

**Comment by freq-mod** [comment]
OK
Synthesis Type 2 : AM instead of FM
Level of operator 1 : 54 - 56 instead of 59

That sounds good enough for me

**Comment by sneakernets** [comment]
aaah okay I'll look at this! I didn't see this until you closed it.. that's weird!

**Comment by freq-mod** [comment]
> I didn't see this until you closed it.. that's weird!

(ס_ס;;)

anyway, the GENMIDI.wopl I attached in comment for 820a0bb contains this "improved" rock organ
still doesn't sound right, but it doesn't resemble weird mix of strings and accordion

**Comment by sneakernets** [comment]
Yeah Github seems to think I care about stuff I starred years ago instead of my own projects, which is annoying!!!
